recipes/emcee/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.1.0" %}
+
+package:
+  name: emcee
+  version: {{ version }}
+
+source:
+  fn: emcee-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/de/00/2358f12c98fa74ab58d78e6a4a4f9a8152fadf6ade2d809d98d771dcc31b/emcee-{{ version }}.tar.gz
+  md5: c6b6fad05c824d40671d4a4fc58dfff7 
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
```

Comment:
GitHub also shows no C or C++ code, which I find to be a good indicator too. As we are not using Cython here, that also helps to confirm this.